orga/tests/integration/components/campaign/activity/delete-participation-modal_test.js

```diff
@@ -46,19 +46,19 @@ module('Integration | Component | Campaign::Activity::DeleteParticipationModal',
           lastName: 'La frite',
           htmlSafe: true,
         });
-        assert
-          .dom(
-            screen.getByText((content, node) => {
-              const hasText = (node) => node.innerHTML.trim() === expectedText.__string;
-              const nodeHasText = hasText(node);
-              const childrenDontHaveText = Array.from(node.children).every((child) => !hasText(child));
-              return nodeHasText && childrenDontHaveText;
-            }),
-          )
-          .exists();
-        assert.contains(this.intl.t('pages.campaign-activity.delete-participation-modal.text'));
-        assert.contains(this.intl.t('pages.campaign-activity.delete-participation-modal.actions.cancel'));
-        assert.contains(this.intl.t('pages.campaign-activity.delete-participation-modal.actions.confirmation'));
+        assert.ok(
+          screen.getByText((content, node) => {
+            const hasText = (node) => node.innerHTML.trim() === expectedText.__string;
+            const nodeHasText = hasText(node);
+            const childrenDontHaveText = Array.from(node.children).every((child) => !hasText(child));
+            return nodeHasText && childrenDontHaveText;
+          }),
+        );
```

Comment:
C'était une sacrée fonction 😅 mais comme on est sur un titre, je propose : 

```suggestion
        assert.ok(screen.getByRole('heading', { name: 'Supprimer la participation de Joe La frite ?' }));
```